admin/includes/functions/html_output.php

```diff
@@ -113,13 +113,14 @@ function zen_image_submit($image, $alt = '', $parameters = '') {
 ////
 // Draw a 1 pixel black line
   function zen_black_line() {
-    return zen_image(DIR_WS_IMAGES . 'pixel_black.gif', '', '100%', '1');
+    return zen_image(DIR_WS_IMAGES . 'pixel_black.gif', '', '', '1', 'style="width:100%;"');
   }
 
 ////
 // Output a separator either through whitespace, or with an image
   function zen_draw_separator($image = 'pixel_black.gif', $width = '100%', $height = '1') {
-    return zen_image(DIR_WS_IMAGES . $image, '', $width, $height);
+      if (substr(rtrim($width), -1) != "%") $width = $width . 'px';
+    return zen_image(DIR_WS_IMAGES . $image, '', '', $height, 'style="width:' . $width . ';"');

```

Comment:
I think when using style, width should also say px after the value

Reply:
for HTML5 only pixels are allowed, so no units are needed.
Other pages are calling this function with either a percentage or with no unit at all, so making the change in the function means the calling pages do not need to change.

Reply:
ok, wasn't sure so that's why I asked